### DISCUSSION TITLE
Layout

### DISCUSSION THREAD

**Discussion by nglidden** [IntelliTect/EssentialCSharp.Web#9] [Feedback and Ideas]
Is it possible to make the sidebar with the ToC collapsable? It all feels very squished side-to-side and top-to-bottom. There's almost an overload of information. 
![c#](https://user-images.githubusercontent.com/88691056/188696415-c833bcbc-c45f-42da-85ed-385f47df6389.png)

The first paragraph of each section is also not indented, while the remaining paragraphs are. Is this a style choice? 

Images could also be bigger. 

There also seem to be roughly up to five different fonts on a page. Somewhat jarring for the reader.

The sizing of the headers doesn't follow the typical article formatting style recommendations. 
![c#4](https://user-images.githubusercontent.com/88691056/188697244-182c9672-a75d-4311-b055-cdaa8242f161.png)

The font size of bullet points is smaller than the font size for the corresponding section. 
![c#2](https://user-images.githubusercontent.com/88691056/188696780-e20e9213-d61f-413c-a537-aa6e9c1f4952.png)

The right margin is very tight.
![c#3](https://user-images.githubusercontent.com/88691056/188697080-133e5e3c-c8e7-48f8-a3db-7ab1baad71c5.png)

**Comment by BenjaminMichaelis** [comment]
Thank you for all this feedback, this is very helpful and we will try to keep this conversation updated as things are implemented to fix.

> Is it possible to make the sidebar with the ToC collapsable? It all feels very squished side-to-side and top-to-bottom. There's almost an overload of information.

Thank you! Fixing!

> The first paragraph of each section is also not indented, while the remaining paragraphs are. Is this a style choice?

Yes, this is a style choice, but given this feedback we will only indent when it is not after a heading for consistency.

> Images could also be bigger.

Agree, will look at increasing the sizes generally.

> There also seem to be roughly up to five different fonts on a page. Somewhat jarring for the reader.

We will look at reducing the number of fonts used, this is helpful input.

> The sizing of the headers doesn't follow the typical article formatting style recommendations.

What is the sizing supposed to follow/ what is that supposed to look like?.

> The font size of bullet points is smaller than the font size for the corresponding section.

This appears to be a bug, we will work on fixing this.

> The right margin is very tight. 

We will look on increasing the margins on larger screens.

**Comment by MarkMichaelis** [comment]
@nglidden: Thanks for the wonderful feedback. This is very helpful as we you have documented first impressions which none of us have.  Thanks so much for taking the time.
